paddle/fluid/operators/multinomial_op.cu

```diff
@@ -31,6 +32,16 @@ __global__ void NormalizeProbability(T* norm_probs, const T* in_data,
                                      T* sum_rows) {
   int id = threadIdx.x + blockIdx.x * blockDim.x +
            blockIdx.y * gridDim.x * blockDim.x;
+  PADDLE_ENFORCE(
+      in_data[id] >= 0.0,
+      "The input of multinomial distribution should be >= 0, but got %f",
+      in_data[id]);
+  PADDLE_ENFORCE(in_data[id] != INFINITY,
```

Comment:
Is there any special reason that checking INF/NaN is added here? Otherwise, I think it is not really necessary. Because the property that a number is not NAN or INF should be satisfied almost everywhere, and if we check it everywhere, it may slow down the system.

Reply:
have removed checking INF/NaN